### ISSUE TITLE
预测出现全黑问题

### ISSUE THREAD

**Issue by wzr0108** [bubbliiiing/deeplabv3-plus-pytorch#15]
用voc训练集训练出来的模型，预测时全黑
`# ---------------------------------------------------#
#   取出每一个像素点的种类
# ---------------------------------------------------#
pr = pr.argmax(axis=-1)`

这个pr是全0的，用预训练模型继续训练，也会出现黑点越来越多的情况，请问这是什么原因呢

**Comment by wzr0108** [comment]
自己训练100轮再进行正向传播
model.train()的时候，不会全0
model.eval()的时候，会全0
但是用你的预训练模型就是正常的

**Comment by bubbliiiing** [comment]
嗯？看一下注释，标签有问题

**Comment by wzr0108** [comment]
标签应该没问题啊，我看了DeeplabDataset返回的png
![屏幕截图 2022-02-25 112039](https://user-images.githubusercontent.com/98294772/155647691-1e4bc7fc-4014-4c24-b30f-0ba2e445b227.png)
这是每一类对应的像素点的个数，是正常的吧

**Comment by zysc1996** [comment]
我也遇到了这个问题，训练是正常的，预测突然就全是黑的了

**Comment by ZhouChen-start** [comment]
> 我也遇到了这个问题，训练是正常的，预测突然就全是黑的了

兄弟解决了吗？我也是训练的时候计算的指标好好的，每一轮验证的时候损失就炸掉了

**Comment by ZhouChen-start** [comment]
> 我也遇到了这个问题，训练是正常的，预测突然就全是黑的了
兄弟解决了吗？我也是训练的时候计算的指标好好的，每一轮验证的时候损失就炸掉了，用的是
deeplabV3_plus_xception，deeplabV3_plus_mobilenet就没这个问题
